### PR TITLE
Remove usage of globals in ilChatroomSettingsGUI

### PR DESCRIPTION
**Pull request by legionth**
Related to https://www.ilias.de/mantis/view.php?id=22851

### CODE REVIEW

**Comment by mjansenDatabay** [comment]
#22851

File: Modules/Chatroom/classes/gui/class.ilChatroomSettingsGUI.php [hunk 1]
```diff
@@ -30,11 +30,12 @@ public function __construct(ilChatroomObjectGUI $gui)
 	 */
 	public function saveGeneral()
 	{
-		/**
-		 * @var $ilCtrl ilCtrl
-		 * @var $lng    ilLanguage
-		 */
-		global $ilCtrl, $lng;
+		global $DIC;
```
Comment (mjansenDatabay) [inline Modules/Chatroom/classes/gui/class.ilChatroomSettingsGUI.php]: Please check if you can use ...

```php
$this->ilCtrl       = $DIC->ctrl();
$this->ilLng       = $DIC->language();
```
... provided by the base class.

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]

**Comment by legionth** [comment]
Implemented your suggestions @mjansenDatabay . Have a look

**Comment by legionth** [comment]
Rebased the branch. And added implemented your suggestions. Have a look @mjansenDatabay

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**